Dockerfile.assisted-test-infra

```diff
@@ -53,7 +59,9 @@ ENV PATH=$PATH:/usr/local/go/bin:/go/bin
 
 COPY . .
 
-RUN chgrp -R 0 /home/assisted-test-infra && \
+# init terraform in order to cache the provider
+RUN terraform -chdir=/home/assisted-test-infra/terraform_files/equinix-ci-machine init && \
```

Comment:
I don't know if there is a better solution, it's required to have the provider already in place when destroying the machines using ansible.